scripts/classes/app-configurator.js

```diff
@@ -83,12 +84,14 @@ class AppConfigurator {
 
   prepareExtensions() {
     const extensions = getExtensionsFromConfiguration(this.configuration);
-    const localExtensions = getLocalExtensions(this.buildConfig.workingDirectories);
+    const localExtensions = _.filter(getLocalExtensions(this.buildConfig.linkedExtensions), (localExt) =>
```

Comment:
Extract `const linkedExtensions = getLocalExtensions(this.buildConfig.linkedExtensions);` above to make filter easier to read.